crates/bevy_window/src/window.rs

```diff
@@ -19,10 +19,12 @@ pub struct WindowId(Uuid);
 /// may be observed with `Immediate` mode, but will not be observed with `Mailbox` or
 /// `Fifo`.
 ///
-/// `Immediate` or `Mailbox` will gracefully fallback to `Fifo` when unavailable.
+/// `AutoVsync` or `AutoNoVsync` will gracefully fallback to `Fifo` when unavailable.
 ///
-/// The presentation mode may be declared in the [`WindowDescriptor`](WindowDescriptor::present_mode)
-/// or updated on a [`Window`](Window::set_present_mode).
+/// `Immediate` or `Mailbox` will panic if not supported by the platform.
+///
+/// The presentation mode may be declared in the [`WindowDescriptor`](WindowDescriptor) using [`WindowDescriptor::present_mode`](WindowDescriptor::present_mode)
+/// or updated on a [`Window`](Window) using [`set_present_mode`](Window::set_present_mode).
```

Comment:
Hey, thanks for fixing the documentation. :)

Anyways, I figured I should point out an error I noticed. These doc-comments don't seem to be respecting the 80-character-per-line max that the rest of the doc-comments are following. It's not a huge deal that it doesn't follow the standard set by other doc-comments, it'd just be nice if everything was consistent.

This would need to be fixed in another PR (or in a commit by a member) since this PR is already merged, but I thought it was worth pointing out.

Reply:
Hey thanks for pointing it out! It's my first time doing such a thing so didn't realize that was important. Unless it's fixed otherwise I'll submit another PR to correct it. My apologies!

Reply:
Ah, no need to apologize, or even worry about it. Being a code style issue (and also because it doesn't affect how documentation is generated), I consider it pretty minor in the grand scheme of things. :)

Reply:
doc style is not strictly enforced, and not consistent through the codebase. Personally I most of the time try to stick to under 100 characters. Sometimes 120...

Reply:
Good to know anyways! I'll try and be more mindful of the quality and consistency in the future, thank you guys!